firmware/gameport-adapter/Logitech.h

```diff
@@ -145,6 +180,7 @@ class Logitech : public Joystick {
     uint8_t numSecondaryHats{};
     uint8_t hasHat{};
     uint8_t numHatDirections{};
+    uint8_t isThunder{};
```

Comment:
This is actually not needed. The deviceID == 0x01 marks the ThunderPad Digital. We can add an enum with various types of the devices in this class. Currently only one entry would be sufficient, but may be in the future we will need more.